ConnectivityDemos/ConnectivityDemosTest/src_test/rest/bpm/TestDataMapping.java

```diff
@@ -11,16 +11,22 @@
 
 import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
 import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmElement;
+import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
 import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
 
 
 @IvyProcessTest
 public class TestDataMapping
 {
+  private static final BpmProcess ODATA_PROCESS = BpmProcess.name("odata");
+  private static final BpmProcess OPEN_API_PROCESS = BpmProcess.name("openapi");
+  
   @Test
   public void odataJsonMapping_single(BpmClient bpmClient) throws Exception
   {
-    ExecutionResult result = bpmClient.start().process("rest/odata/readById.ivp").execute();
+    BpmElement readById = ODATA_PROCESS.elementName("readById.ivp");
```

Comment:
In my opinion we have changed here an expressive string to a less expressive API. 
While I love the process selector when it comes to mocking - I prefer the other for starts.
Changing all the examples to the BpmProcess like approch makes the other valid setup less approachable for new users. 
So I'd have kept them as is ... and only use it in mocking samples. Or when mocking and starts can benefit from using the same selectors. But not in this test here.

Reply:
Ok